api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -3,261 +3,325 @@
+          width: 600px;
+          border-radius: 8px;
+          background-color: rgba(33,111,180,0.1);
+          padding-left: 1.5rem;
+          padding-top:1rem
+      }
+      .stepCard {
+          height: 126px;
+          width: 580px;
+          border-radius: 8px;
+          background-color: #FFFFFF;
+      }
+      .step_rt_ct {
+          height: auto;
+          padding-top: 0.1rem
+      }
+      .step3 {
+          height: auto;
+          width: 513px;
+          padding-bottom: 0.5rem;
+          padding-left: 40px;
+      }
+      .completeRegistrationBtn {
+          /*padding-left: 140px;*/
+          padding-bottom: 1rem;
+      }
+      .user-info {
+          font-weight: bold;
+          display: inline-block;
+          margin-left: 6px;
+          padding-top: 0.1rem;
+      }
+      .bold-user-label {
+          font-weight: bold;
+          display: inline-block;
+      }
+      .card {
+          line-height: 22px;
+          margin-bottom: 27px;
+          text-decoration: none;
+      }
 
-    .bullet img {
-      display: inline-block;
-      height: 31px;
-      width: 31px;
-      margin-left: 1rem;
-      margin-right: 1rem;
-      border-radius: 15px;
-      background-color: #302C71;
-      color: #FFFFFF;
-      vertical-align: middle;
-    }
+      .secondTitle {
+          font-size: 18px;
+          padding-top: 0.7rem;
+          font-weight: bold
+      }
 
-    .bullet {
-      padding-top: .5em;
-    }
+      .bullet img {
+          display: inline-block;
+          height: 31px;
+          width: 31px;
+          margin-left: 1rem;
+          margin-right: 1rem;
+          border-radius: 15px;
+          background-color: #302C71;
+          color: #FFFFFF;
+          vertical-align: middle;
+      }
 
-    .button {
-      height: 3.25rem;
-      width: 16rem;
-      border-radius: 5px;
-      background-color: #302C71;
-      font-size: 16px;
-      font-weight: 500;
-      color: #FFFFFF !important;
-      display: inline-block;
-      justify-content: space-around;
-      align-items: center;
-      text-align: center;
-      line-height: 52px;
-      text-transform: uppercase;
-      text-decoration: none;
-      cursor: pointer;
-    }
-    th.short {
-      width: 20%;
-    }
-    th.long {
-      width: 60%
-    }
-    a.text-link {
-      text-decoration: none;
-      color: #2691D0;
-    }
+      .bullet {
+          padding-top: .5em;
+      }
+
+      .button {
+          height: 3.25rem;
+          width: 16rem;
+          border-radius: 5px;
+          background-color: #302C71;
+          font-size: 16px;
+          font-weight: 500;
+          color: #FFFFFF !important;
+          display: inline-block;
+          justify-content: space-around;
+          align-items: center;
+          text-align: center;
+          line-height: 52px;
+          text-transform: uppercase;
+          text-decoration: none;
+          cursor: pointer;
+      }
+      th.short {
```

Comment:
Are th.short and th.long needed? It looks like they are only used with the `&nbsp;` in the thead.

It looks like td.shortlabel and td.longtext are not being used.

Reply:
Yes, i noticed some styles left back. i am formatting the html as well so that its easier to read